src/scss/components/_cards.scss

```diff
@@ -62,6 +62,20 @@ $card-section-collapsed-height--aboveMid        : 50px;
         padding-right: spacing(x2);
     }
 
+        .c-card--negativeSpacing--belowMid {
+            @include media('<mid') {
+                margin-left: -#{spacing(x2)};
```

Comment:
Do you need the `#{...}` ?

Reply:
Yes, otherwise the compiled CSS comes out as `margin-left: -spacing();`. It needs the interpolation in order to compile the correct value with the minus sign 🐱

Reply:
If you put a space between the minus and function it works. But not a biggy here, just good to know, the interpolation is mostly best used for multiplication and adding variable and numbers together. :)

Reply:
Adding parenthesis around it works as well (`margin-left: (-spacing(x2));`).